src/plugins/expression_reveal_image/common/i18n/ui/dict/reveal_image.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import { i18n } from '@kbn/i18n';
+
+export const strings = {
```

Comment:
nit: I see it was in the previous implementation, but it looks like an overhead to me. Finally we use that translations from one place.  + I see only `canvas` use that pattern. Maybe we should  discuss it now

Reply:
Ok, I'm not against it)